README.md

```diff
@@ -434,7 +434,10 @@ On MacOS
 - AppleClang 11.0
 
 On Windows
-- Visual Studio 2017
+- Visual Studio 2019
+
+  You might still build using 2017, but its c++17 support is incomplete so
```

Comment:
This could be shorter. We already say "Other compilers or later versions may work, but they are not tested." on the next line.  This could simply say:

> Visual Studio 2017 does not have full C++17 support, and testing for it may be stopped soon.

Reply:
Yes, that's better.